### PR TITLE
Adding GA Analytics and DAP code.

### LINKED ISSUE #6: Add DAP code and Google Analytics

### PR DESCRIPTION
**Pull request by juliasolorzano**
Fixes #6

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -2,6 +2,19 @@
 <html lang="en">
   <head>
     {{ partial "head.html" . }}
+    
+    <script>
```
Comment (rogeruiz) [inline layouts/index.html]: Should we add this to the `head.html` partial so we can include it on other pages, _as they come down the pipe_. Or is the initialization on `L12-15` custom to the homepage?

File: layouts/index.html [hunk 2]
```diff
@@ -143,5 +156,8 @@ <h3 class="usa-font-lead">All our work is guided by user research — structured
     <!--Footer links END -->
 
     <script src="assets/scripts/main.js"></script>
+
```
Comment (rogeruiz) [inline layouts/index.html]: With two scripts, is it time to create a `script.html` partial? The `index.html` file only gets used for the `/` route and needs to be ported over to `layouts/_default/single.html` eventually for any single pages.

File: layouts/index.html (hunk 1 above)
Comment (juliasolorzano) [inline layouts/index.html]: Fixed here a00867a3a664d3c5947767ea5c98eba926109721

File: layouts/index.html (hunk 2 above)
Comment (juliasolorzano) [inline layouts/index.html]: Added a `scripts.html` partial here  aaf7df7

**Comment by rogeruiz** [comment]
LGTM, :+1:

**Pull request merged**